codec/protobuf/protobuf_test.go

```diff
@@ -2,18 +2,27 @@ package protobuf
 
 import (
 	"testing"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 
-	"github.com/golang/protobuf/proto"
+	"github.com/asdine/storm"
 	"github.com/asdine/storm/codec/internal"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestProtobuf(t *testing.T) {
-	u1 := SimpleUser{
-		Id: proto.Uint64(1),
-		Name: proto.String("John"),
-	}
+	u1 := SimpleUser{Id: 1, Name: "John"}
 	u2 := SimpleUser{}
 	internal.RoundtripTester(t, Codec, &u1, &u2)
-	assert.True(t, u1.GetId() == u2.GetId())
+	assert.True(t, u1.Id == u2.Id)
+}
+
+func TestSave(t *testing.T) {
+	dir, _ := ioutil.TempDir(os.TempDir(), "storm")
+	defer os.RemoveAll(dir)
+	db, _ := storm.Open(filepath.Join(dir, "storm.db"), storm.Codec(Codec))
+	u := SimpleUser{Id: 1, Name: "John"}
+	err := db.Save(&u)
```

Comment:
For this change I would also read the user back from the db and make sure they are the same.